lib/asciidoctor/csa/converter.rb

```diff
@@ -1,19 +1,22 @@
 require "asciidoctor"
-require "metanorma/csand/version"
-require "isodoc/csand/html_convert"
+require "metanorma/csa/version"
+require "isodoc/csa/html_convert"
+require "isodoc/csa/pdf_convert"
+require "isodoc/csa/word_convert"
 require "asciidoctor/standoc/converter"
 require "fileutils"
 require_relative "./validate.rb"
 
 module Asciidoctor
-  module Csand
-        CSAND_NAMESPACE = "https://open.ribose.com/standards/csand"
+  module Csa
+    CSA_NAMESPACE = "https://open.ribose.com/standards/csand"
```

Comment:
@ronaldtse @opoudjis I didn;t changed XML namespace to save backward compatibility, make sense?

Reply:
@CAMOBAP795 let's change it as well.